lib/dalli/protocol/meta.rb

```diff
@@ -28,6 +28,7 @@ def response_processor
       def write_multi_storage_req(_mode, pairs, ttl = nil, _cas = nil, options = {})
         ttl = TtlSanitizer.sanitize(ttl) if ttl
 
+        fire_forget = options&.key?(:fire_forget) ? options.delete(:fire_forget) : false
```

Comment:
this is to test out an idea we had for a nice speedup, but will need to be tested with a custom version of internal deployment...